src/main/webapp/i18n/en/student-dashboard.json

```diff
@@ -14,7 +14,7 @@
                 "registerSuccessful": "Register successful",
                 "signUpForCourse": "Sign up for selected course",
                 "cancel": "Cancel",
-                "noCourse": "No course available to register to",
+                "noCourse": "No courses available to sign up for",
                 "courseLoading": "Courses are loading...",
                 "pleaseSelectCourse": "# Select a course",
                 "courseSignupConfirmationTitle": "Confirm Course Registration"
```

Comment:
We should use either _register_ or _sign up_ consistently.

Reply:
I agree, consistency could be improved upon. In that specific context I think both are viable options; "register" would be more formal but "sign up" is somehow friendlier. "Enroll" is another option we might want to consider. Does anyone else have any thoughts? I'll keep considering this in the meantime.